reactor-core/src/main/java/reactor/core/publisher/MonoCreate.java

```diff
@@ -156,6 +156,9 @@ public void success(@Nullable T value) {
 							actual.onNext(value);
 							actual.onComplete();
 						}
+						catch (Throwable t) {
+							actual.onError(t);
```

Comment:
at first I was wondering about calling `error` here, but this reproduces the gist of the method, which wouldn't work anyway due to the `STATE` guard. The outer `finally` should correctly trigger `disposeResource` anyway, so :+1:

Reply:
Catching errors in `MonoDelayUntil` solves the actual issue by calling `onError` internally. In this cases `catch` in  `MonoCreate` is never reached. However, it ensures sending `error` signal in such cases. Right?

Reply:
yeah I think the two fixes are complementary